### PR TITLE
[NEWDEV] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: dll/win32/newdev/lang/ro-RO.rc [hunk 1]
```diff
@@ -15,12 +15,12 @@ CAPTION "Expert de instalare dispozitiv"
 FONT 8, "MS Shell Dlg"
 BEGIN
     LTEXT "Bun venit!", IDC_WELCOMETITLE, 120, 8, 195, 16
-    LTEXT "Acest expert va instala un nou driver de dispozitiv pentru:", IDC_STATIC, 120, 21, 195, 16
-    LTEXT "Apăsați Înainte pentru a continua.", IDC_STATIC, 120, 169, 195, 16
+    LTEXT "Expertul facilitează instalarea de produse software pentru:", IDC_STATIC, 120, 21, 195, 16
```
Comment (GeoB99) [inline dll/win32/newdev/lang/ro-RO.rc:18]: Why "produse software"? This interprets as any software that could be installed, a game is also a "produs". The previous translation was fine.

Comment (AndreiMiloiu) [reply dll/win32/newdev/lang/ro-RO.rc:18]: You're right. I'll revert it.

Comment (AndreiMiloiu) [reply dll/win32/newdev/lang/ro-RO.rc]: Fixed!

**Comment by AndreiMiloiu** [comment]
Any news?

**Pull request merged**